generators/app/index.js

```diff
@@ -1,489 +1,30 @@
 'use strict'
-const path = require('path')
+
 const {default: chalk} = require('chalk')
 const yosay = require('yosay')
-const inquirer = require('inquirer')
-const Generator = require('../../lib/generator')
-const validator = require('./validator')
+const Generator = require('yeoman-generator')
 
 module.exports = class extends Generator {
-  constructor(args, opts) {
```

Comment:
This file was moved and the named re-used so all the deleted lines from this show up in `generators/node/index.js`.